### PR TITLE
Feat/client/column

### PR DESCRIPTION
**Pull request by jhaemin**
### Prepend placeholder (DND)

`appendChild` appends an element to the end. According to the HTML hierarchy, next elements are placed above the previous elements. So, prepend the placeholder instead of append, since the placeholder should always be placed under all other cards.

### Page styles

* make the column relative
* increase zIndex of column header
* apply the top and bottom spaces to the cards container itself
* add the placeholder style

### CODE REVIEW

File: src/client/scripts/column.ts [hunk 1]
```diff
@@ -1,11 +1,31 @@
...
 
-if (newColumnBtn) {
-  newColumnBtn.addEventListener('click', async () => {
-    const columnHtml = `
+function onClickNewColumnBtn(newColumnBtn: HTMLElement) {
+  const columnHtml = `
 <div class="column">
   <div class="column-header">
-    <h2 class="column-name">untitled column</h2>
+    <h2 class="column-name" spellcheck>untitled column</h2>
```
Comment (dnacu) [inline src/client/scripts/column.ts:28]: false를 해주세요!

File: src/client/scripts/column.ts [hunk 2]
```diff
@@ -19,59 +39,117 @@ if (newColumnBtn) {
...
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({
-        // TODO: default to what?
-        name: 'untitled column',
-        // TODO: dynamically get the previous column id
-        previousColumnId: 1,
-      }),
-    }).then((res) => {
+  fetch('/board/1', {
```
Comment (dnacu) [inline src/client/scripts/column.ts:51]: div.app의 data-board-id에 있어요!

File: src/client/scripts/drag-and-drop.ts [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+  })
+
+  let pmc: (e: PointerEvent) => void
+
+  window.addEventListener(
+    'pointermove',
+    (pmc = (e) => {
+      // Calculate the delta X, Y
+      const deltaX = e.pageX - initialX
+      const offsetY = e.pageY - initialY
```
Comment (dnacu) [inline src/client/scripts/drag-and-drop.ts:132]: 일관성을 유지해주세요.

File: src/client/scripts/column.ts (hunk 1 above)
Comment (jhaemin) [reply src/client/scripts/column.ts:28]: 완료!

**Pull request merged**